pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -232,6 +234,15 @@ var _ = Describe("ImageUpload", func() {
 		Expect(err).ToNot(HaveOccurred())
 	}
 
+	createDataSource := func(ds *cdiv1.DataSource) {
+		defer GinkgoRecover()
+		time.Sleep(10 * time.Millisecond)
+
+		// TODO: Fix fake client
```

Comment:
What is this about?

Reply:
Forgot to update it to use fake client, `cdiClient`.